HerPublicWebsite/Startup.cs

```diff
@@ -95,6 +95,8 @@ public void ConfigureServices(IServiceCollection services)
                 setup.SchemaName = configuration.GetSection("SessionCache")["SchemaName"];
             });
 
+            services.AddMvc().AddSessionStateTempDataProvider();
+

```

Comment:
We store the addresses returned by OSPlaces in TempData. When OSPlaces returned 100 results the temp data cookie got too large and broke the site. This stores TempData in the session data in the DB.